infra/scripts/test-end-to-end-batch-dataflow.sh

```diff
@@ -0,0 +1,269 @@
+    export feast_kafka_2_ip=$(gcloud compute addresses describe ${ip_addr_name} --region=asia-east1 --format "value(address)")
+  elif [[ "$ip_count" == 2 ]]; then
+    export feast_kafka_3_ip=$(gcloud compute addresses describe ${ip_addr_name} --region=asia-east1 --format "value(address)")
+  elif [[ "$ip_count" == 3 ]]; then
+    export feast_redis_ip=$(gcloud compute addresses describe ${ip_addr_name} --region=asia-east1 --format "value(address)")
+  elif [[ "$ip_count" == 4 ]]; then
+    export feast_statsd_ip=$(gcloud compute addresses describe ${ip_addr_name} --region=asia-east1 --format "value(address)")
+  fi
+  ip_count=$((ip_count + 1))
+  export "$(echo $ip_addr_name | tr '-' '_')=$(gcloud compute addresses describe ${ip_addr_name} --region=asia-east1 --format "value(address)")"
+done
+
+echo "
+============================================================
+Creating GKE nodepool for Feast e2e test with DataflowRunner
+============================================================
+"
+gcloud container clusters create ${K8_CLUSTER_NAME} --region ${GCLOUD_REGION} \
+    --enable-cloud-logging \
+    --enable-cloud-monitoring \
+    --network ${GCLOUD_NETWORK} \
+    --subnetwork ${GCLOUD_SUBNET} \
+    --machine-type n1-standard-2
+sleep 120
+
+echo "
+============================================================
+Create feast-postgres-database Secret in GKE nodepool
+============================================================
+"
+kubectl create secret generic feast-postgresql --from-literal=postgresql-password=password
+
+echo "
+============================================================
+Create feast-gcp-service-account Secret in GKE nodepool
+============================================================
+"
+cd $ORIGINAL_DIR/infra/scripts
+kubectl create secret generic feast-gcp-service-account --from-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+echo "
+============================================================
+Export required environment variables
+============================================================
+"
+export TEMP_BUCKET=$TEMP_BUCKET
+export DATASET_NAME=$DATASET_NAME
+export GCLOUD_PROJECT=$GCLOUD_PROJECT
+export GCLOUD_NETWORK=$GCLOUD_NETWORK
+export GCLOUD_SUBNET=$GCLOUD_SUBNET
+export GCLOUD_REGION=$GCLOUD_REGION
+
+echo "
+============================================================
+Helm install Feast and its dependencies
+============================================================
+"
+cd $ORIGINAL_DIR/infra/scripts/test-templates
+envsubst $'$TEMP_BUCKET $DATASET_NAME $GCLOUD_PROJECT $GCLOUD_NETWORK \
+  $GCLOUD_SUBNET $GCLOUD_REGION $feast_kafka_1_ip
+  $feast_kafka_2_ip $feast_kafka_3_ip $feast_redis_ip $feast_statsd_ip' < values-end-to-end-batch-dataflow.yaml > $ORIGINAL_DIR/infra/charts/feast/values-end-to-end-batch-dataflow-updated.yaml
+
+cd $ORIGINAL_DIR/infra/charts/feast
+helm install --wait --timeout 600s --values="values-end-to-end-batch-dataflow-updated.yaml" ${HELM_RELEASE_NAME} .
```

Comment:
Would be great if you could add a helm template command before this for visibility!

Reply:
Added! Let me know if its better now.